docs/architecture/index.md

```diff
@@ -58,6 +58,10 @@ Request timeout for icmp_seq 150
 
 Within a Kubernetes cluster, the load-balancing is managed by the `plndr-cloud-provider` which watches all service that are created, and for those of `type: LoadBalancer` will create the configuration for `kube-vip` to consume.
 
+#### Load Balancing (Inside a cluster)
+
+When using `type:LoadBalancer` within a Kubernetes cluster `kube-vip` will assign the VIP to the leader (when using ARP) all to all running Pods (when using BGP). When traffic is directed to a node with the VIP then the rules configured by `kube-proxy` will redirect the traffic to one of the pods running in the service.
```

Comment:
I think this is usually written is `type=LoadBalancer`.

In any case, it probably should be, "when creating a Service of type=LoadBalancer" etc.

Reply:
"all to all"?

Reply:
It might help to have an introductory paragraph, explaining that Kubernetes itself takes care of getting traffic directed at a Service IP to the correct Pods, so kube-vip deals solely with getting the traffic onto a cluster node, etc.